internal/testing/require/require.go

```diff
@@ -126,6 +127,19 @@ func Error(t TestingT, err error, formatWithArgs ...interface{}) {
 	}
 }
 
+// EqualErrno should be used for functions that return syscall.Errno or nil.
+func EqualErrno(t TestingT, expected syscall.Errno, err error, formatWithArgs ...interface{}) {
```

Comment:
this is the new assertion, which helps a lot to figure out when different OS choose the same literal errno for different errors!